### DISCUSSION TITLE
Using datatable in conda environment

### DISCUSSION THREAD

**Discussion by eedokl** [h2oai/datatable#2795] [Q&A] [answered]
Hi

I have a problem when I pip install datatable to my conda based windows environment.
Datatable as such works perfectly. However, once installed my conda commands like conda list or conda env export are not working anymore - and it seems related to a SHA256 checksum performed on the License file in the datatable package.

How to reproduce:
after pip install datatable run "conda list"

Debug trace:
...
      File "C:\Program Files\Anaconda3\lib\site-packages\conda\common\pkg_formats\python.py", line 269, in get_paths
        records = process_csv_row(record_reader)
      File "C:\Program Files\Anaconda3\lib\site-packages\conda\common\pkg_formats\python.py", line 249, in process_csv_row
        cleaned_path, checksum)
    **AssertionError: ('C:\\Users\\xyz\\.conda\\envs\\debug\\Lib\\site-packages\\datatable-0.11.0.dist-info', 'Lib/site-packages/datatable-0.11.0.dist-info/LICENSE', 'HyVuytGSiAUQ6ErWBHTqt1iSGHhLmlC8fO7jTCuR8dU')**

Any ideas, or someone out there having seen this as well?

**Comment by st-pasha** [comment, marked as answer]
This should have been fixed in release 0.11.1. Please trying installing it and see if the problem still persists.

**Reply by eedokl** [reply]
Retried and no problem seen anymore. Thanks a lot.
